### PR TITLE
Fix: all - Fix issues for 1.5 RC1

### PR DESCRIPTION
**Pull request by oxedions**
Fix few issues found by users on 1.5 RC1

### CODE REVIEW

**Review by strus38 — APPROVED** [review]

**Comment by strus38** [comment]
approved for the code... if the CI is working of course!

**Comment by oxedions** [comment]
Yes, seems to be an issue with registry. I will investigate asap.

File: CHANGELOG.md [hunk 1]
```diff
@@ -40,6 +40,7 @@
     - notify in readme/page that firewall does not work in chroot (#569)
     - fix python path of diskless files (#590)
   - dns_server:
+    - Add missing vital parameters to allow binding to external DNS servers (#XXX)
```
Comment (neilmunday) [inline CHANGELOG.md:43]: I take it the issue number will be added here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -48,13 +49,15 @@
     - allow custom configuration path (#591)
     - remote logs stored in a similar fashion as the server (#568)
   - nic_nmcli:
+    - Add reboot capability, needed on some system. (#XXX)
```
Comment (neilmunday) [inline CHANGELOG.md:52]: I take it the issue number will be added here?

File: CHANGELOG.md [hunk 3]
```diff
@@ -48,13 +49,15 @@
...
     - add ip4_manual entry (#618)
     - add dns4 and dns4_search vars logic (#585)
     - improve role capabilities (#558)
     - add missing register of nic_nmcli_apply variable (#662)
   - powerman:
     - implement support for externaly defined BMC (#640)
   - pxe_stack:
+    - Fix missing tftp path for Ubuntu (#XXX)
```
Comment (neilmunday) [inline CHANGELOG.md:60]: I take it the issue number will be added here?

File: roles/core/nic_nmcli/readme.rst [hunk 4]
```diff
@@ -175,14 +175,17 @@ time set their configuration.
 However, in some cases, users might need to force some updates (for example if 
 you wish to set routes on the main interface, etc).
 
-To achieve that, two variables are at disposal:
+To achieve that, few variables are at disposal:
```
Comment (neilmunday) [inline roles/core/nic_nmcli/readme.rst:178]: Perhaps reword toL

_To achieve this, a few variables are at your disposal:_

Comment (oxedions) [reply roles/core/nic_nmcli/readme.rst:178]: Thanks for correction, done :-)

**Comment by oxedions** [comment]
Issues are related to no more working auto-tests. I will merge for now, and I will work on auto-tests in the next release. This will be the first task.

**Pull request merged**